src/bluehawk/commands/SnippetCommand.ts

```diff
@@ -84,14 +85,17 @@ export const SnippetCommand = makeBlockCommand<IdRequiredAttributes>({
     // Dedent
     dedentRange(clonedSnippet, commandNode);
 
+    // ID is required, so it should exist
```

Comment:
I would note in this comment that it's checked in the validator before being passed here, hence assert instead of throw.